File: Principles of Programming Language/HW5P/Arrays.py
import random
print("Welcome to the Arrays Program in Python")
print("Written by Matt McCullough")
print("With assistance from Andy Mac")
size = int(input("Enter a value: "))

randomlist = []
for i in range(0,size):
    n = random.randint(10,99)
    randomlist.append(n)
print(randomlist)

File: Principles of Programming Language/HW5J/Arrays.java
import java.util.Random;
import java.util.Scanner;

public class Arrays {
    public static void main(String[] args)
    {
        System.out.println("Welcome to the Array Program in Java");
        System.out.println("Written by Matt McCullough");
        System.out.println("With assistance from Andy Mac");

        Scanner reader = new Scanner(System.in);
        System.out.print("Enter a value: ");
        int size = reader.nextInt();

        int[] array = new int[size];
        Random rng = new Random();

        for(int i=0; i<array.length; i++)
        {
            array[i] = rng.nextInt(90)+10;
        }
        System.out.println("The contents of the array are: ");
        for(int i=0; i<array.length; i++)
        {
            System.out.println(array[i]);
        }


    }
}
